src/main/java/duke/Event.java

```diff
@@ -0,0 +1,28 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task{
```

Comment:
Perhaps  adding a space after Task would be more consistent with your other class declarations.

Reply:
Thanks, I have fixed it!